src/status_im/utils/instabug.cljs

```diff
@@ -53,3 +53,6 @@
                    config/instabug-token
                    (.. rn-dependencies/instabug -invocationEvent -shake))
   (.setIntroMessageEnabled rn-dependencies/instabug false))
+
+(defn report-js-exception [exception]
+  (.reportJSException rn-dependencies/instabug exception))
```

Comment:
there is shorter `def instabug` already defined in this ns so you can use it instead

Reply:
ok